### ISSUE TITLE
Copy / past error

### ISSUE THREAD

**Issue by faheemhameed** [glideapps/glide-data-grid#381]
Hi there,

When I copy cells using Command+C I got the following error: (on my testing server - Apache)

```
2main.db4f79d7.js:2 Uncaught (in promise) TypeError: Cannot read properties of undefined (reading 'write')
    at nY (main.db4f79d7.js:2:1667363)
    at main.db4f79d7.js:2:1705409
    at d (main.db4f79d7.js:2:672275)
    at Generator._invoke (main.db4f79d7.js:2:672028)
    at Generator.next (main.db4f79d7.js:2:672638)
    at r (main.db4f79d7.js:2:668272)
    at s (main.db4f79d7.js:2:668475)
    at main.db4f79d7.js:2:668534
    at new Promise (<anonymous>)
    at main.db4f79d7.js:2:668415
nY @ main.db4f79d7.js:2
(anonymous) @ main.db4f79d7.js:2
d @ main.db4f79d7.js:2
(anonymous) @ main.db4f79d7.js:2
(anonymous) @ main.db4f79d7.js:2
r @ main.db4f79d7.js:2
s @ main.db4f79d7.js:2
(anonymous) @ main.db4f79d7.js:2
(anonymous) @ main.db4f79d7.js:2
(anonymous) @ main.db4f79d7.js:2
o @ main.db4f79d7.js:2
```

With the same code base, the Copy / paste works fine when I run it on my local machine (react dev server & local Apache after code was built)

Any lights on this would make me move forward.

Many thanks.

**Comment by jassmith** [comment]
What are you using to bundle? Find the mangled code thats crashing and share the snippet around it? I might be able to identify it.

**Comment by faheemhameed** [comment]
I am using 

cross-env GENERATE_SOURCEMAP=false react-app-rewired build

to build.

I will try to provide more info.

But the strange thing is it works locally under the apache server (Mac OS) but doesn't work in the CentOS test server.

Thanks.

**Comment by faheemhameed** [comment]
After kind of not disabling the source map generation, I got this error.

```
data-editor-fns.js:269 Uncaught (in promise) TypeError: Cannot read properties of undefined (reading 'write')
    at nY (data-editor-fns.js:269:34)
    at data-editor.js:1866:9
    at d (regeneratorRuntime.js:86:17)
    at Generator._invoke (regeneratorRuntime.js:66:24)
    at Generator.next (regeneratorRuntime.js:117:21)
    at r (asyncToGenerator.js:3:20)
    at s (asyncToGenerator.js:25:9)
    at asyncToGenerator.js:32:7
    at new Promise (<anonymous>)
    at asyncToGenerator.js:21:12
```

Also I am using this grid editor def.

```
        <DataEditor
          getCellContent={getCellContent}
          ref={dataEditor}
          columns={columns}
          headerHeight={50}
          rowHeight={50}
          rows={gridRowsLength}
          height={windowHeight - global.elementsHeightWithoutListGrid}
          onColumnResize={onColumnResize}
          onVisibleRegionChanged={onVisibleRegionChanged}
          onCellClicked={onCellClicked}
          getCellsForSelection={true}
        />
```

**Comment by jassmith** [comment]
seems you have a browser whose navigation.clipboard is undefined. Fixed 4.1.1

**Comment by faheemhameed** [comment]
Thanks. I will check it out.
